src/util/error.cpp

```diff
@@ -45,6 +45,11 @@ bilingual_str ResolveErrMsg(const std::string& optname, const std::string& strBi
     return strprintf(_("Cannot resolve -%s address: '%s'"), optname, strBind);
 }
 
+bilingual_str InvalidPortErrMsg(const std::string& optname, const std::string& strPort)
```

Comment:
nit: Just call it `port`, or maybe `port_str`; we don't use Hungarian variable names in new code.

Reply:
Actually, something like `invalid_value` might be better, since this *isn't* just a bare port number.

Reply:
Thanks again for your input, Luke